app/services/audit/warc_auditer.rb

```diff
@@ -35,8 +35,12 @@ def audit
     attr_reader :collection_druid, :wasapi_collection_id, :wasapi_account, :wasapi_provider, :embargo_months
 
     def wasapi_warc_filenames
-      WasapiWarcLister.new(wasapi_collection_id: wasapi_collection_id, wasapi_provider: wasapi_provider,
-                           wasapi_account: wasapi_account, embargo_months: embargo_months).to_a
+      WasapiWarcLister.new(
+        wasapi_collection_id: wasapi_collection_id,
+        wasapi_provider: wasapi_provider,
+        wasapi_account: wasapi_account,
+        embargo_months: embargo_months
+      ).pluck(:filename)
```

Comment:
Even better than the `map`!

Reply:
rubocop suggested :pluck to me for a migrator class in DSA -- it never would have occurred to me.

Reply:
@ndushay I got it from rubocop as well! I much prefer these curated nudges to AI driven code complete.